sceval_frontend/src/containers/Hardware.tsx

```diff
@@ -49,32 +49,19 @@ const Hardware = withRouter((props: HardwareProps & RouteComponentProps) => {
             let deviceBundles = linkedModules;
             deviceResponse.forEach((device: any, index: any) => {
               // for each device, set linked modules
-              const url = `${MODE_API_BASE_URL}devices/${device.id}/kv?keyPrefix=sensorModule`;
               setIsLoading(true);
-              modeAPI
-                .request('GET', url, {})
-                .then((sensorModules: any) => {
-                  const filteredModules = sensorModules.data.filter(
```

Comment:
This is removed because we don't need to do this anymore. getAllDeviceKeyValueStoreByPrefix will return only key/value stores that have keys starts with "sensorModule"